Add changelog entries for 11.0.0_beta2 (#5895)
citusdata/citus: CHANGELOG.md

> ### citus v11.0.0_beta2 (April 4, 2022) ###

Reviewer: ```suggestion
### citus v11.0.1_beta (April 11, 2022) ###
```

> * Adds support for `DOMAIN` objects

Reviewer: ```suggestion
* Adds support for propagation of `DOMAIN` objects
```

Author: I guess we have a better alternative if we want to use the word `propagation` here.

How about

```suggestion
* Adds propagation of `DOMAIN` objects
```

This looks similar to the items below

```
* Adds propagation for foreign server commands

* Adds propagation of `TEXT SEARCH CONFIGURATION` objects

* Adds propagation of `TEXT SEARCH DICTIONARY` objects

...

* Adds propagation of `CREATE SCHEMA .. GRANT ON SCHEMA ..` commands

...

* Adds propagation of `SCHEMA` operations
```

* Adds support for `TABLESAMPLE`

* Allows adding a unique constraint with an index

* Fixes a bug that could cause `EXPLAIN ANALYZE` to fail for prepared statements
  with custom type

* Fixes a bug that could cause Citus not to create function in transaction block
  properly

> * Fixes a bug that could cause invalid JSON when running `EXPLAIN ANALYZE` with

Reviewer: ```suggestion
* Fixes a bug that could cause returning invalid JSON when running `EXPLAIN ANALYZE` with
```

> * Fixes a bug that could cause returning invalid JSON when running
> `EXPLAIN ANALYZE` with subplans

Reviewer: I guess we were aligning lines in such cases ?

```suggestion
* Fixes a bug that could cause returning invalid JSON when running
  `EXPLAIN ANALYZE` with subplans
```

> * Fixes a bug that prevents non-client backends to access shards

Reviewer: ```suggestion
* Fixes a bug that prevents non-client backends from accessing shards
```
